service/dynamodb/expression/doc.go

```diff
@@ -0,0 +1,49 @@
+/*
+Package expression provides types and functions to create DynamoDB Expression
+strings, ExpressionAttributeNames maps, and ExpressionAttributeValues maps.
+
+Using the Package
+
+The package represents the various DynamoDB Expressions as builder structs named
+accordingly. For example, ConditionBuilder represents a DynamoDB Condition
+Expression, an UpdateBuilder represents a DynamoDB Update Expression, and so
+forth.
+The following example will show a sample ConditionExpression and how to build an
+equilvalent ConditionBuilder
+
+  // Let :a be an ExpressionAttributeValue representing the string "No One You
+  // Know"
+  condExpr := "Artist = :a"
+  condBuilder := expression.Name("Artist").Equal(expression.Value("No One You Know"))
+
+In order to retrieve the formatted DynamoDB Expression strings, users must call
```

Comment:
nit, audience is the reader, not a third party the reader will instruct on how to use the library.

In this case can replace `users must call`, with just `call the`.